src/test/resources/features/accounts/navigateToEenadu.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Navigating to eenadu paper home page
```

Comment:
Can you move this feature file into somewhere else?
I am assuming this branch is for travel scenarios.

Reply:
Done